File: src/main/java/org/intellij/sdk/language/TweeTokenType.java
package org.intellij.sdk.language;

import com.intellij.psi.tree.IElementType;
import com.intellij.psi.tree.xml.IXmlLeafElementType;

public interface TweeTokenType {
    IElementType TWEE_START_MACRO_START = new IXmlLeafElementType("TWEE_START_MACRO_START");
    IElementType TWEE_MACRO_END = new IXmlLeafElementType("TWEE_MACRO_END");
}

File: src/main/kotlin/org/intellij/sdk/language/psi/TweeTokenType.kt
package org.intellij.sdk.language.psi

import com.intellij.psi.tree.IElementType
import org.intellij.sdk.language.TweeLanguage
import org.jetbrains.annotations.NonNls

class TweeTokenType(@NonNls debugName: String) : IElementType(debugName, TweeLanguage) {
    override fun toString(): String {
        return "TweeTokenType.${super.toString()}"
    }
}
